.github/workflows/.reusable-docker-build.yml

```diff
@@ -0,0 +1,84 @@
+# reusable workflow
+name: Build Docker Image
+
+on:
+  workflow_call:
+    inputs:
+      registry-url:
+        type: string
+        description: Github container registry base URL
+        required: false
+        default: ghcr.io
+      file:
```

Comment:
`file` is pretty ambiguous here. Is there a reason not to use `dockerfile` ?

Reply:
I wanted to use the same argument names as the build action does to keep minimal cognitive overhead.